operator/tests/controller/metrics/100-rbac.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: operator-metrics-unprivileged
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: operator-metrics-privileged
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+# We cannot simply apply a ClusterRoleBinding resource, because of the variable $NAMESPACE
```

Comment:
Out of curiosity. Is it a general issue with kuttl that you have to create cluster scoped resources this way, or did you just want to add $NAMESPACE to the name to be sure you don't have naming conflicts?

Reply:
Yes, in general we'd want to avoid conflicts on re-runs, etc.

But additionally this particular case is a little special, because it contains namespace name twice - once in the name, and also its body contains a reference to a namespaced service account, and this reference has to include a namespace name.